### PR TITLE
Dev/oculus api140

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Updated Oculus to use the new 1.40 release, API's now included with package as Unity is too far behind

Also updated package for UPM delivery and upped versions.

Also added functions to get / set the tracking origin to the OculusAPI scope

## Changes:

- Fixes: #13
- Fixes: #15

### CODE REVIEW

File: XRTK.Oculus/Packages/manifest.json [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+        "com.xrtk"
+      ]
+    }
+  ],
   "dependencies": {
     "com.unity.package-manager-ui": "2.1.2",
     "com.unity.textmeshpro": "2.0.1",
-    "com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git#0.1.16",
-    "com.xrtk.upm-git-extension": "https://github.com/XRTK/UpmGitExtension.git#0.9.6",
+    "com.xrtk.core": "0.1.19",
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/manifest.json]: let's add the sdk as well

Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/manifest.json]: I've added it as requested, but none of the other extensions are including SDK

File: XRTK.Oculus/Packages/com.xrtk.oculus/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "com.xrtk.oculus",
   "displayName": "XRTK.Oculus",
   "description": "The Oculus platform components for the Mixed Reality Toolkit",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "unity": "2019.1",
   "license": "MIT",
   "repository": {
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/package.json]: Merge in the changes from #14 or update the package.json

Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/package.json]: Done

**Pull request merged**